### ISSUE TITLE
Fail on EE 2.5.2 install

### ISSUE THREAD

**Issue by rgbink** [andrewfairlie/rc_debug#1]
What's the oldest supported version of EE? Fails with:

```
Error Number: 1146
Table 'p4p.exp_template_routes' doesn't exist
SELECT `route`, `template_id` FROM (`exp_template_routes`) WHERE `route` IS NOT NULL
Filename: third_party/rc_debug/pi.rc_debug.php
Line Number: 275
```

Should add the version dependency to the Readme.

**Comment by andrewfairlie** [comment]
Agreed. This plugin was a proof of concept idea that we never really continued with. Still open to the idea though.

The truth is that I don't know, but as it's looking for Template Routes I imagine 2.8 (when they were introduced).

If you're not on 2.8 you could delete that part from the plugin.

There's really not been that much demand for this type of plugin for us to continue with it really, but feel free to fork and develop as you see fit.

**Comment by rgbink** [comment]
Works for me. Thanks.
